[2023-12-27T04:06:41.095+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: potential_customers_process.create_customers_table manual__2023-12-27T04:06:38.729617+00:00 [queued]>
[2023-12-27T04:06:41.107+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: potential_customers_process.create_customers_table manual__2023-12-27T04:06:38.729617+00:00 [queued]>
[2023-12-27T04:06:41.107+0000] {taskinstance.py:1362} INFO - 
--------------------------------------------------------------------------------
[2023-12-27T04:06:41.107+0000] {taskinstance.py:1363} INFO - Starting attempt 1 of 1
[2023-12-27T04:06:41.108+0000] {taskinstance.py:1364} INFO - 
--------------------------------------------------------------------------------
[2023-12-27T04:06:41.121+0000] {taskinstance.py:1383} INFO - Executing <Task(PostgresOperator): create_customers_table> on 2023-12-27 04:06:38.729617+00:00
[2023-12-27T04:06:41.149+0000] {standard_task_runner.py:55} INFO - Started process 723 to run task
[2023-12-27T04:06:41.154+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'potential_customers_process', 'create_customers_table', 'manual__2023-12-27T04:06:38.729617+00:00', '--job-id', '18', '--raw', '--subdir', 'DAGS_FOLDER/potential_customers_process.py', '--cfg-path', '/tmp/tmp73819bvf']
[2023-12-27T04:06:41.155+0000] {standard_task_runner.py:83} INFO - Job 18: Subtask create_customers_table
[2023-12-27T04:06:41.279+0000] {task_command.py:376} INFO - Running <TaskInstance: potential_customers_process.create_customers_table manual__2023-12-27T04:06:38.729617+00:00 [running]> on host bbee0f1e1252
[2023-12-27T04:06:41.617+0000] {taskinstance.py:1592} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=potential_customers_process
AIRFLOW_CTX_TASK_ID=create_customers_table
AIRFLOW_CTX_EXECUTION_DATE=2023-12-27T04:06:38.729617+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-12-27T04:06:38.729617+00:00
[2023-12-27T04:06:41.638+0000] {base.py:71} INFO - Using connection ID 'postgres' for task execution.
[2023-12-27T04:06:41.649+0000] {sql.py:315} INFO - Running statement: 
            CREATE TABLE IF NOT EXISTS customers (
                fullname TEXT NOT NULL,
                country_code TEXT NOT NULL,
                gender TEXT NOT NULL,
                age INT NOT NULL,
                last_login INT NOT NULL
            );
        , parameters: None
[2023-12-27T04:06:41.651+0000] {postgres.py:96} INFO - NOTICE:  relation "customers" already exists, skipping

[2023-12-27T04:06:41.666+0000] {taskinstance.py:1406} INFO - Marking task as SUCCESS. dag_id=potential_customers_process, task_id=create_customers_table, execution_date=20231227T040638, start_date=20231227T040641, end_date=20231227T040641
[2023-12-27T04:06:41.711+0000] {local_task_job.py:164} INFO - Task exited with return code 0
[2023-12-27T04:06:41.747+0000] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
